HoneyDo.Web/ClientApp/package.json

```diff
@@ -9,18 +9,12 @@
     "react-dom": "^16.6.0",
     "react-router-dom": "^4.3.1",
     "react-scripts": "^2.1.1",
-    "react-styleguidist": "^8.0.2",
     "rimraf": "^2.6.2"
   },
   "devDependencies": {
-    "eslint": "^5.8.0",
```

Comment:
I removed eslint as a dependency, however as you found out react scripts has it included so we still have it, other eslint related dependencies were removed because we can’t use your config, which used those dependencies, with react scripts.  I don’t know what “storybook” means in the context of react.

Reply:
I removed eslint as a dependency, however as you found out react scripts has it included so we still have it, other eslint related dependencies were removed because we can’t use your config, which used those dependencies, with react scripts.  I don’t know what “storybook” means in the context of react.